templates/rustdoc/topbar.html

```diff
@@ -193,7 +193,11 @@
         {%- endif -%}
 
         <li class="pure-menu-item">
-            <a href="{{ latest_path | safe }}" class="pure-menu-link warn"
+            <a href="/crate/{{ metadata.name }}/latest/target-redirect/
+                {%- if target != "" %}{{target}}{% else %}{{ krate.metadata.default_target }}/{% endif -%}
+                {{ inner_path }}
+                {%- if query %}?{{query}}{% endif -%}
```

Comment:
Doing this in the template looks relevant prone to error. Why did you chose to do it here instead of modifying latest_path?

Reply:
At first I thought it would be simple, and then I kept running into complications as I ran the tests. This does belong in the Rust code.

I wanted to avoid modifying latest_path because latest_path is now used by the permalink. But I think what we need is, e.g. latest_path and permalink_path, both generated by Rust.